StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1613ElementParameterDocumentationMustDeclareParameterName.cs

```diff
@@ -50,7 +56,36 @@ public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics
         /// <inheritdoc/>
         public override void Initialize(AnalysisContext context)
         {
-            // TODO: Implement analysis
+            context.RegisterSyntaxNodeAction(this.HandleElement, SyntaxKind.MethodDeclaration);
+        }
+
+        private void HandleElement(SyntaxNodeAnalysisContext context)
+        {
+            var node = context.Node;
+
+            var documentation = XmlCommentHelper.GetDocumentationStructure(node);
+
+            if (documentation != null)
+            {
+                if (XmlCommentHelper.GetTopLevelElement(documentation, XmlCommentHelper.InheritdocXmlTag) != null)
```

Comment:
:exclamation: It seems we *don't* want to skip these comments for this analyzer.